src/Sylius/Behat/Page/Product/ProductShowPage.php

```diff
@@ -20,13 +19,13 @@
 class ProductShowPage extends SymfonyPage
 {
     /**
-     * @param ProductInterface $product
+     * @param array $urlParameters
      *
      * @return ProductShowPage
      */
-    public function openSpecificProductPage(ProductInterface $product)
+    public function open(array $urlParameters = [])
```

Comment:
The `open()` wasn't opening a page and it was the only exception from the rule. If we manage to specify pages as a services, every page will be able to define their own opening methods.